packages/react-components/review/beta/react-components.api.md

```diff
@@ -1251,6 +1220,55 @@ export const _LocalVideoTile: React_2.MemoExoticComponent<(props: {
     personaMinSize?: number | undefined;
 }) => JSX.Element>;
 
+// @beta
+export interface Mention {
+    displayText: string;
+    icon?: JSX.Element;
+    id: string;
+}
+
+// @beta
+export interface MentionDisplayOptions {
+    onRenderMention?: (mention: Mention) => JSX.Element;
+}
+
+// @beta
+export interface MentionLookupOptions {
+    onQueryUpdated: (query: string) => Promise<Mention[]>;
+    onRenderSuggestionItem?: (suggestion: Mention, onSuggestionSelected: (suggestion: Mention) => void) => JSX.Element;
+    trigger?: string;
+}
+
+// @beta
+export type MentionOptions = {
+    lookupOptions?: MentionLookupOptions;
+    displayOptions?: MentionDisplayOptions;
+};
+
+// @internal
+export const _MentionPopover: (props: _MentionPopoverProps) => JSX.Element;
+
+// @internal
+export interface _MentionPopoverProps {
```

Comment:
We set this as an internal props, which means we are not exposing it to customer right now
Any plans on exposing the popover publicly or we will always keep it internal as part of MessageThread

Reply:
Current plan is internal only.

Reply:
> We set this as an internal props, which means we are not exposing it to customer right now Any plans on exposing the popover publicly or we will always keep it internal as part of MessageThread

The popover will be internal until there's some customer asks.